branchpro/simulation.py

```diff
@@ -64,6 +64,18 @@ def switch_resolution(self, num_points):
         self._regime = np.rint(np.linspace(
             start_sim_time, end_sim_time, num=num_points)).astype(int)
 
+    def get_regime(self):
+        """
+        Gets all time point the simulation uses.
+        """
+        return self._regime
+
+    def get_time_bounds(self):
+        """
+        Gets time bounds of the simulation.
```

Comment:
I would remark on the type that gets returned since it's not obvious (i.e., tuple with 2 elements)